component/component.js

```diff
@@ -67,6 +67,13 @@ export default Ember.Component.extend(NodeDriver, {
       this.set('model.%%DRIVERNAME%%Config.networks', [])
     }
 
+    let labels_list = this.get('model.%%DRIVERNAME%%Config.serverLabel');
```

Comment:
```
    if (!this.get('model.%%DRIVERNAME%%Config.networks')) {
      this.set('model.%%DRIVERNAME%%Config.networks', [])
    }
```

shouldn't it be something like that enough? (did you test it?)

Reply:
Yes also works in my tests. 

I testes both. With and without labels. Actually created a server. No errors in any of the logs.